setup/linux/build_deb.sh

```diff
@@ -24,7 +24,7 @@ PACKAGE_NAME="mmex-$MMEX_VERSION-$ARCHITECTURE"
 cd ../..
 MMEX_DIR=`pwd`
 
-./bootstrap
+./bootstrap.sh
```

Comment:
I've renamed bootstrap to sh since it is a sh file and the .gitattributes file will cover it then without adding a new rule